easybuild/tools/options.py

```diff
@@ -568,6 +588,13 @@ def _postprocess_list_avail(self):
             self.log.info(msg)
         else:
             print msg
+
+        # cleanup tmpdir
+        try:
+            shutil.rmtree(self.options.tmpdir)
+        except OSError as err:
+            raise EasyBuildError("Failed to clean up temporary directory %s: %s", self.options.tmpdir, err)
```

Comment:
If this goes wrong, something is very wrong, you'd want to know (a warning would only pop up in the log, so it's very likely you'll miss it).